doc/dune-files.rst

```diff
@@ -1310,6 +1314,10 @@ running dune runtest you can use the following stanza:
     (libraries alcotest mylib)
     (action (run %{test} -e)))
 
+Starting from dune 2.8, test modules without interface files (e.g. `.mli`,
```

Comment:
I'd say "Starting from (lang dune 2.8)" rather than dune 2.8 to convey that it's opt-in (not sure if there's already similar language in the docs)

Reply:
`(lang dune 2.8)` does seem better to me; I was trying to be consistent with the existing language in the docs (e.g. https://dune.readthedocs.io/en/stable/dune-files.html#explicit-js-mode) . Happy to change some or all of the occurrences as you prefer.